src/api/app/services/workflows/scm_event_subscription_creator.rb

```diff
@@ -1,16 +1,26 @@
 module Workflows
   class ScmEventSubscriptionCreator
-    def initialize(token, workflow_run, scm_webhook, package)
+    def initialize(token, workflow_run, scm_webhook, object_for_subscription_association)
```

Comment:
Can you stick to one? Either calling it `object_for_...` or `class_a_or_class_b`

Reply:
Sure, I stick now to the `class_a_or_class_b`